src/test/java/com/breskul/bibernate/persistence/EntityManagerImplTest.java

```diff
@@ -0,0 +1,57 @@
+package com.breskul.bibernate.persistence;
+
+import com.breskul.bibernate.configuration.PersistenceProperties;
+import com.breskul.bibernate.exeptions.JdbcDaoException;
+import com.breskul.bibernate.persistence.testModel.Person;
+import com.breskul.bibernate.repository.DataSourceFactory;
+import jakarta.persistence.EntityManager;
+import org.junit.jupiter.api.*;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.function.Consumer;
+
+@TestClassOrder(ClassOrderer.OrderAnnotation.class)
+public class EntityManagerImplTest {
```

Comment:
You can extends from AbstractDataSourceTest and use already created data source 
![image](https://user-images.githubusercontent.com/32467464/228062415-3a0fff60-3a2f-4ae9-9a73-589fb7645dd5.png)